### PR TITLE
Fix deploy

### PR DESCRIPTION
**Pull request by willronchetti**
- Reverts accession code computation, which is currently broken

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ chardet = "3.0.4"
 colorama = "0.3.3"
 dcicpyvcf = "1.0.0"
 dcicsnovault = "4.5.0"
-dcicutils = "^1.10.0"
+dcicutils = "1.11.0"
```
Comment (netsettler) [inline pyproject.toml:47]: `^1.11.0` ?

File: src/encoded/schema_formats.py [hunk 2]
```diff
@@ -16,23 +16,22 @@
...
-                raise RuntimeError("accession_type in %s is not a 2-character string:", letter_pair)
-            letter_pairs.add(letter_pair)
-    return "|".join(sorted(letter_pairs))
-
-
-ACCESSION_CODES = compute_accession_codes()
-
-accession_re = re.compile(r'^%s(%s)[1-9A-Z]{7}$' % (ACCESSION_PREFIX, ACCESSION_CODES))
-test_accession_re = re.compile(r'^%s(%s)[0-9]{4}([0-9][0-9][0-9]|[A-Z][A-Z][A-Z])$' % (TEST_PREFIX, ACCESSION_CODES))
+# XXX: This is broken, cannot decode schema files - Will 3/2/2021
```
Comment (netsettler) [inline src/encoded/schema_formats.py:19]: It troubles me that no tests caught this problem.

**Review by netsettler — APPROVED** [review]
Yes, I am fine with this.

File: src/encoded/schema_formats.py (hunk 2 above)
Comment (willronchetti) [reply src/encoded/schema_formats.py:19]: I'm guessing the encoding is resolved correctly on the local machine as `make deploy1` works. There is clearly something different on the Beanstalk I'd say.

**Pull request merged**